code/game/gamemodes/miniantags/demons/shadow_demon/shadow_demon.dm

```diff
@@ -57,6 +59,33 @@
 	max_integrity = 100
 	light_color = "#AAD84B"
 	anchored = TRUE
+	var/time_since_last_hallucination = 0
```

Comment:
can you document this variable, `time` doesn't actually refer to actual `world.time` here and actually represents cycles since last hallucination so that needs to be clarified

Reply:
Done